src/test/java/competition/subsystems/turret/TurretSubsystemTest.java

```diff
@@ -0,0 +1,19 @@
+package competition.subsystems.turret;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import competition.BaseCompetitionTest;
+
+public class TurretSubsystemTest extends BaseCompetitionTest {
+
+    @Test
+    public void testStop() {
+        TurretSubsystem turret = this.injector.getInstance(TurretSubsystem.class);
+        turret.stop();
+
+        assertEquals(0 , turret.motor.getMotorOutputPercent(), 0.001);
```

Comment:
This isn't a great test because even if turret.stop() didn't do anything at all it would still pass (because the getMotorOutputPercent() was going to give you 0 by default no matter what). 

Consider setting the turret motor to be not stopped before you call `stop()`